### DISCUSSION TITLE
PDF Tags

### DISCUSSION THREAD

**Discussion by drfurtado** [quarto-dev/quarto-cli#5669] [Q&A]
I teach with the LMS Canvas. Every time I upload a PDF that was rendered in Quarto, Canvas Accessibility score is really bad. Upon checking, it says that the PDF is untagged. Is there a function in Quarto that would create a `tagged` PDF when rendered?

Then it explains...

`Most documents are created in a word-processing program such as Word or PowerPoint before they are converted to a PDF. Many programs create PDF files, but few produce structured or tagged PDFs. PDF tags are hidden labels that clarify the structure of the document (e.g., table, heading, paragraph, etc.). Untagged PDFs do not contain any of this information and can cause the content to be misinterpreted.`

**Comment by baptiste** [comment]
You could try to follow this: https://libguides.lib.msu.edu/c.php?g=995742&p=8207771

I haven't tried those packages myself, but it might make sense to have something like this in Quarto, since other formats already make use of alt information in figures, etc.
